resources/views/posts/completed.blade.php

```diff
@@ -0,0 +1,19 @@
+@extends('layout')
+
+@section('title')
+    Created a new post!
+@endsection
+
+@section('content')
+    Thanks for using Edmonton's Location Integrated Messaging and Experience Service! <BR>
+    Your post will appear on this buddy bench shortly after moderator approval. <BR>
+    We hope you have a nice day! <br>
+
+    @if($post->hasImage())
+        <img src="{{$post->getImage()}}"> <br>
+    @endif
+
+
+    <!-- Include some limes here -->
+    <a href="{{url()->current()}}"> Back to location page </a>
+@endsection
```

Comment:
Use `@stop` instead of `@endsection`

No functionality change, just the preferred dialect.

Also if you're using PHPstorm or whatever editor configure it to add newlines at the end of files.